### PR TITLE
Clean up SVN properties post-migration to Git

### PR DESCRIPTION
**Pull request by shaoguangleo**
This commit removes SVN-related properties and metadata that were leftover after migrating the source code from SVN to Git. The cleanup includes removing the `SVN properties (DO NOT CHANGE) ` section mainly in the front of the source code that is no longer needed.

The purpose of this cleanup is to streamline the repository and avoid potential conflicts or confusion arising from residual SVN artifacts in a Git environment.

As this cleanup includes many source code files, although I thought it would not affect our toolchain, I also need you to review this.

### CODE REVIEW

**Comment by gbcrew** [comment]
certainly seems sensible to me.  However I'm not in a position to evaluate for several weeks.

File: libraries/difxmessage/utils/statemon [hunk 1]
```diff
@@ -19,6 +19,7 @@
 #   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
 #**************************************************************************
 
+
```
Comment (adamdeller) [inline libraries/difxmessage/utils/statemon:22]: One blank line was inadvertently added here (just proving that I was paying attention!)

File: sites/MPIfR/mark5/m5fixtrack/m5fixtrack.c [hunk 2]
```diff
@@ -16,17 +16,7 @@
...
-//
-// $Id: m5fixtrack.c 5105 2013-01-08 09:27:22Z HelgeRottmann $
-// $HeadURL: https://svn.atnf.csiro.au/difx/sites/MPIfR/mark5/m5fixtrack/m5fixtrack.c $
-// $LastChangedRevision: 5105 $
-// $Author: HelgeRottmann $
-// $LastChangedDate: 2013-01-08 17:27:22 +0800 (二, 2013-01-08) $
-//
-//============================================================================
-//
+
```
Comment (adamdeller) [inline sites/MPIfR/mark5/m5fixtrack/m5fixtrack.c:19]: And a blank line here, although I guess this is more in keeping with the usual format

File: mpifxcorr/utils/genmachines.py [hunk 3]
```diff
@@ -18,16 +18,7 @@
...
-# SVN properties (DO NOT CHANGE)
-#
-# $Id$
-# $HeadURL: $
-# $LastChangedRevision$
-# $Author$
-# $LastChangedDate$
-#
-#============================================================================
+
```
Comment (adamdeller) [inline mpifxcorr/utils/genmachines.py:21]: This one could be deleted

File: libraries/difxmessage/utils/restartdifx [hunk 4]
```diff
@@ -1,5 +1,6 @@
 #!/bin/env python3
 
+
```
Comment (adamdeller) [inline libraries/difxmessage/utils/restartdifx:3]: blank line added

**Review by adamdeller — APPROVED** [review]
Looks all fine to me (if you wan to be really tidy, there were a few added blank lines that can be removed)

**Comment by shaoguangleo** [comment]
> Looks all fine to me (if you wan to be really tidy, there were a few added blank lines that can be removed)

@adamdeller Thanks for the review.
Everything looks good to me as well. I have also resolved some conflicts. 
If everything looks fine, we can go ahead and merge the PR to the `dev` branch.

**Review by jwagnerhki — APPROVED** [review]

**Comment by shaoguangleo** [comment]
@adamdeller @jwagnerhki  Thanks for the review, I think I can merge and close them.

**Pull request merged**